Framework/MDAlgorithms/test/IntegratePeaksMD2Test.h

```diff
@@ -180,6 +180,119 @@ class IntegratePeaksMD2Test : public CxxTest::TestSuite {
     TS_ASSERT(algF.isExecuted());
   }
 
+  //-------------------------------------------------------------------------------
+  /** Setup simple algorithm*/
+  static void simpleRun(bool shouldPass, size_t numberValuesPeakRadius, size_t numberValuesBkgInnerRadius,
+                        size_t numberValuesBkgOuterRadius, bool cyl = false, bool ellip = false) {
+    createMDEW();
+    // --- Make a fake PeaksWorkspace ---
+    PeaksWorkspace_sptr peakWS(new PeaksWorkspace());
```

Comment:
Use
`auto peakWs = std::make_shared<PeakWorkspace>();`

instead of bare new, see:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rr-make_shared

Reply:
Guilty of copying code, tried to change all occurrences of new!